client/state/action-types.js

```diff
@@ -81,6 +81,8 @@ export const BILLING_TRANSACTIONS_REQUEST_SUCCESS = 'BILLING_TRANSACTIONS_REQUES
 export const CONCIERGE_AVAILABLE_TIMES_REQUEST = 'CONCIERGE_AVAILABLE_TIMES_REQUEST';
 export const CONCIERGE_AVAILABLE_TIMES_UPDATE = 'CONCIERGE_AVAILABLE_TIMES_UPDATE';
 export const CONCIERGE_APPOINTMENT_CANCEL = 'CONCIERGE_APPOINTMENT_CANCEL';
+export const CONCIERGE_APPOINTMENT_DETAILS_REQUEST = 'CONCIERGE_APPOINTMENT_DETAILS_REQUEST';
+export const CONCIERGE_APPOINTMENT_DETAILS_UPDATE = 'CONCIERGE_APPOINTMENT_DETAILS_UPDATE';
```

Comment:
[Please keep this list alphabetized](https://github.com/Automattic/wp-calypso/blob/cd31a84b049a07a5d7c7e41f0114b2ed5409e0f7/client/state/action-types.js#L6) :trollface:

Reply:
Right, updated.